specs/protocol/predeploys.md

```diff
@@ -37,9 +37,9 @@ Predeploys are used instead of precompiles to make it easier for multiclient
 implementations as well as allowing for more integration with hardhat/foundry
 network forking.
 
-Predeploy addresses exist in 1 byte namespace `0x42000000000000000000000000000000000000xx`.
-Proxies are set at each possible predeploy address except for the
-`GovernanceToken` and the `ProxyAdmin`.
+Predeploy addresses exist in a prefixed namespace `0x420xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx`.
```

Comment:
Don't you think that we should only use `x` in the least significant 3 nibbles to represent the fact that they are the only things that can change?

Reply:
[4f2b9a3](https://github.com/ethereum-optimism/specs/pull/339/commits/4f2b9a332350d79b4f25e643fc247a0078439bd3)